### PR TITLE
Sync menu theme with menu-items by default

### LINKED ISSUE #861: Sync menu theme with menu-items by default
Now we are to explicitly set `menu-item_theme` even though it's already defined for `menu`. I think we can propagate it if there's no modifier `theme` in `ctx`.

// @mishanga

### PR DESCRIPTION
**Pull request by mishanga**
Close #861.
It's already done:
https://github.com/bem/bem-components/blob/ba59063/common.blocks/menu/menu.tmpl-specs/10-default.bemjson.js
https://github.com/bem/bem-components/blob/ba59063/common.blocks/menu/menu.tmpl-specs/10-default.html

I've just did some refactoring.

### CODE REVIEW

**Comment by mishanga** [comment]
@tadatuta

File: common.blocks/menu-item/menu-item.bemhtml [hunk 1]
```diff
@@ -2,5 +2,10 @@ block('menu-item')(
     js()(function() {
         return { val : this.ctx.val };
     }),
-    attrs()({ role : 'menuitem' })
+    attrs()({ role : 'menuitem' }),
+    match(this._menuMods).def()(function() {
+        this.mods.theme = this._menuMods.theme;
```
Comment (tadatuta) [inline common.blocks/menu-item/menu-item.bemhtml]: `var mods = this.mods;`

File: common.blocks/menu-item/menu-item.bemhtml [hunk 2]
```diff
@@ -2,5 +2,11 @@ block('menu-item')(
     js()(function() {
         return { val : this.ctx.val };
     }),
-    attrs()({ role : 'menuitem' })
+    attrs()({ role : 'menuitem' }),
+    match(this._menuMods).def()(function() {
+        var mods = this.mods;
```
Comment (mishanga) [inline common.blocks/menu-item/menu-item.bemhtml:7]: @tadatuta Like this?

Comment (tadatuta) [inline common.blocks/menu-item/menu-item.bemhtml:7]: yes.

:ship:

**Pull request merged**